src/scripts/main.js

```diff
@@ -1 +1,23 @@
 'use strict';
+
+const largeImg = document.querySelector('.gallery__large-img');
+const thumbs = document.querySelector('.gallery__list');
+
+thumbs.addEventListener('click', (e) => {
```

Comment:
Consider using event delegation more effectively by checking if `e.target` is an `a` element directly, rather than checking for an `IMG` and then finding the closest `a`. This could simplify the logic and potentially improve performance.